extensions/asana/src/api/teams.ts

```diff
@@ -0,0 +1,16 @@
+import { request } from "./request";
+
+export type Teams = {
+  gid: string;
+  name: string;
+};
+
+export async function getTeamsForWorkspace(workspace: 1201518352966289) {
```

Comment:
I'm not sure the hardcoded number is supposed to be there.

Reply:
oop! yeah my bad there.